packages/manager/modules/veeam-enterprise/src/dashboard/license/license.controller.js

```diff
@@ -1,8 +1,11 @@
 export default class VeeamEnterpriseLicenseCtrl {
   /* @ngInject */
   constructor(VeeamEnterpriseService) {
+    this.VEEAM_PORT = 9392;
```

Comment:
**suggestion:** Move this to a const at the top of the file, to make it more explicit

Reply:
Have added the a constant file itself. Cleaner